swagger/routes/userRegister.js

```diff
@@ -0,0 +1,15 @@
+const Joi = require('joi');
+
+const registerSwagger = {
+  responses: {
```

Comment:
We can add this part of your route validation only. No need of separate file altogether